docs/src/develop/components/StatusDoc.js

```diff
@@ -24,6 +24,8 @@ var StatusDoc = React.createClass({
             <dd>Smaller sized version.</dd>
             <dt><code>value       error|warning|ok|disabled|unknown</code></dt>
             <dd>Which status to indicate.</dd>
+            <dt><code>a11yTitle  {"{title}"}</code></dt>
+            <dd>Accessibility Title. If not set uses the value default title for status icon.</dd>
```

Comment:
Can we say: 'Accessibility Title. If not set uses the default title for the status icon.'